Packs/RiskIQDigitalFootprint/Integrations/RiskIQDigitalFootprint/RiskIQDigitalFootprint.yml

```
@@ -0,0 +1,3729 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: RiskIQDigitalFootprint
+  version: -1
+configuration:
+- defaultvalue: https://api.riskiq.net
+  display: API URL
+  name: url
+  required: true
+  type: 0
+- display: API Token
+  name: token
+  required: true
+  type: 4
+- display: API Secret
+  name: secret
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: All the RiskIQ Digital Footprint API calls would timeout if the
+    response is not returned within the configured time interval.
+  defaultvalue: '40'
+  display: HTTP(S) Request Timeout (in seconds)
+  hidden: false
+  name: request_timeout
+  required: false
+  type: 0
+- additionalinfo: The maximum number of records to fetch.
+  defaultvalue: '2000'
+  display: Maximum records
+  hidden: false
+  name: max_records
+  required: false
+  type: 0
```

```suggestion
- additionalinfo: All the RiskIQ Digital Footprint API calls would timeout if the
    response is not returned within the configured time interval.
  defaultvalue: '40'
  display: HTTP(S) Request Timeout (in seconds)
  hidden: false
  name: request_timeout
  required: false
  type: 0
- additionalinfo: The maximum number of records to fetch.
  defaultvalue: '2000'
  display: Maximum records
  hidden: false
  name: max_records
  required: false
  type: 0
- display: Trust any certificate (not secure)
  name: insecure
  required: false
  type: 8
- display: Use system proxy settings
  name: proxy
  required: false
  type: 8
```
It's a cleaner look if you have the checkboxes of `proxy` and `insecure` appear at the bottom of the integration configuration panel.

---

Sure, we will make this change.